dds/DCPS/RTPS/RtpsDiscoveryConfig.cpp

```diff
@@ -321,9 +321,12 @@ RtpsDiscoveryConfig::sedp_multicast(bool sm)
 DCPS::NetworkAddress
 RtpsDiscoveryConfig::sedp_local_address() const
 {
+  const DCPS::NetworkAddress d1(TheServiceParticipant->default_address());
+  const DCPS::NetworkAddress d2("0.0.0.0:0");
+
   return TheServiceParticipant->config_store()->get(config_key("SEDP_LOCAL_ADDRESS").c_str(),
-                                                    DCPS::NetworkAddress::default_IPV4,
-                                                    DCPS::ConfigStoreImpl::Format_Required_Port,
+                                                    d1 ? d1 : d2,
```

Comment:
Why not use default_IPv4?

Reply:
Are you asking if `d2` can be initialized to `default_IPV4`?

Reply:
Either that or just use the expression `DCPS::NetworkAddress::default_IPV4` in place of d2 since that's what the code did before this change and it seems to be equivalent.